### PR TITLE
Miscellaneous licensing, metadata, and internationalisation

### PR DESCRIPTION
**Pull request by yell0wfl4sh**
* add .gitignore

* remove MANIFEST,

* fix Python GI API version warnings,

* change `class` to `exec` in metadata,

* add POT file,

* add setup.py.

### CODE REVIEW

**Comment by yell0wfl4sh** [comment]
@quozl added gettext to show.py.

File: show.py [hunk 1]
```diff
@@ -229,7 +230,7 @@ def print_note(msg, msg_color=pause_color):
 
 
 def print_paused():
-    print_note('PAUSED')
+    print_note(_("PAUSED:"))
```
Comment (quozl) [inline show.py]: There's a colon (:) here that isn't in the POT file.

**Comment by yell0wfl4sh** [comment]
Sorry for so many itterations.

**Comment by quozl** [comment]
No worries, it is good to see the improvement in your work.

**Pull request merged**